### DISCUSSION TITLE
Check if named timeseries is already registred in db

### DISCUSSION THREAD

**Discussion by theresaharbom** [ravendb/ravendb#16288] [Q&A] [answered]
Hi!

To register a named timeseries in the store I do:

private struct RoutePoint
{
    [TimeSeriesValue(0)] public double Latitude;
    [TimeSeriesValue(1)] public double Longitude;
}

store.TimeSeries.Register<User, RoutePoint>();

But how can check if a named timeseries is already registred?

**Comment by shiranshalom** [comment, marked as answer]
Hello :)
You can check it by searching the name in `TimeSeriesConfiguration`, if the returned value is not `null` it means a TS with this name already exists:

```
 var timeSeriesConfiguration = store.Maintenance.Server.Send(new GetDatabaseRecordOperation(store.Database)).TimeSeries;
 var routePoint = timeSeriesConfiguration.GetNames("users", nameof(RoutePoint) + "s");
```
